pkg/action/hooks.go

```diff
@@ -79,8 +80,15 @@ func (cfg *Configuration) execHook(rl *release.Release, hook release.HookEvent,
 			return errors.Wrapf(err, "warning: Hook %s %s failed", hook, h.Path)
 		}
 
-		// Watch hook resources until they have completed
-		err = cfg.KubeClient.WatchUntilReady(resources, timeout)
+		// Check if kube.Interface implementation satisfies kube.ContextInterface interface.
+		// If it doesn't log a warning and move on, nothing we can do.
+		if kubeClient, ok := cfg.KubeClient.(kube.ContextInterface); ok {
```

Comment:
If the assertion fails, should WatchUntilReady be used as a fallback? I didn't do this since the idea is Timeout will be replaced completely by the ctx passed into RunWithContext.

Reply:
Updated.